tycho-integration-test/src/stream_processor/protocol_stream_processor.rs

```diff
@@ -76,8 +76,29 @@ impl ProtocolStreamProcessor {
                         continue;
                     }
                 };
-                let update =
-                    StreamUpdate { update_type: UpdateType::Protocol, update, is_first_update };
+                let now = match std::time::SystemTime::now()
+                    .duration_since(std::time::UNIX_EPOCH)
+                    .map(|d| d.as_secs())
```

Comment:
You could keep this as a timestamp for longer, and then substract the seconds from the block, this way you keep things a bit more accurate. I think as_secs just drops the already passed milliseconds so numbers resulting from here are slightly more optimistic.

Reply:
Just curious - is it not possible to save the metric as a float?